src/JS/useFetch.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
```

Comment:
/hooks is a more appropriate directory name. Almost anything could live inside this folder since it's all JS.